src/python/bench.py

```diff
@@ -425,6 +425,67 @@ def run_wiki_1k_schema_bench(start, tgz, runLogDir, perfFile, gcFile):
     finally:
         server.stop()
         time.sleep(5)
+#Same as run_wiki_1k_schema_bench but with overwrite=false 
+#TODO refactor
```

Comment:
Yeah, I think we should just add another overwrite parameter to the existing run_wiki_1k_schema_bench method instead of copying it over.